examples/service-clients/odsp-client/shared-tree-demo/package.json

```diff
@@ -35,14 +35,10 @@
 	"dependencies": {
 		"@azure/msal-browser": "^2.34.0",
 		"@fluid-experimental/odsp-client": "workspace:~",
-		"@fluidframework/container-definitions": "workspace:~",
-		"@fluidframework/container-runtime-definitions": "workspace:~",
-		"@fluidframework/core-interfaces": "workspace:~",
-		"@fluidframework/fluid-static": "workspace:~",
 		"@fluidframework/odsp-doclib-utils": "workspace:~",
 		"@fluidframework/odsp-driver-definitions": "workspace:~",
-		"@fluidframework/tree": "workspace:~",
 		"css-loader": "^1.0.0",
+		"fluid-framework": "workspace:~",
 		"react": "^17.0.1",
```

Comment:
For validation that the trimmed re-exports from 'fluid-framework' are consumable by TSC, I changed 'shared-tree-demo' to import from 'fluid-framework'.